doctors-routes.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+
+from flask import request, Blueprint
+
+from Application.Model.Patients import Patients
+from Application.database import database
+from flask import jsonify
+from flask_cors import cross_origin
+
+doctorapi = Blueprint('doctorapi', __name__)
+
+@cross_origin()
+@app2.route('/doctor/<id_doctor>', methods=["GET", "PUT"])
```

Comment:
route must be defined for doctorapi not for app2. replace app2.route with doctorapi.route